modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -5247,6 +5309,41 @@ private void _assertFilteredObjectEntriesSize(
 			objectEntries.toString(), size, objectEntries.size());
 	}
 
+	private void _assertListTypeEntries(
+		List<Object> expectedListTypeEntries,
+		List<ListEntry> actualListTypeEntries) {
+
+		if (ListUtil.isEmpty(expectedListTypeEntries)) {
+			return;
+		}
+
+		for (Object expectedListTypeEntry : expectedListTypeEntries) {
+			Assert.assertTrue(
+				ListUtil.exists(
+					actualListTypeEntries,
+					listEntry -> {
+						if (expectedListTypeEntry instanceof String) {
+							return Objects.equals(
+								expectedListTypeEntry, listEntry.getKey());
+						}
+
+						return Objects.equals(expectedListTypeEntry, listEntry);
```

Comment:
We could try to reuse the `_assertListTypeEntry()` here :thinking: 
```
actualListTypeEntry -> {
    try {
        _assertListTypeEntry(actualListTypeEntry, expectedListTypeEntry);
        return true; 
    } catch (AssertionError assertionError) {
        return false; 
    }
}
```

Reply:
Done!